enjoy/src/renderer/components/medias/media-current-recording.tsx

```diff
@@ -516,43 +517,42 @@ export const MediaCurrentRecording = () => {
           </DropdownMenuTrigger>
 
           <DropdownMenuContent>
-            {
-              layout?.name === 'sm' && (
-                <>
-                  <DropdownMenuItem
-                    className="cursor-pointer"
-                    onClick={toggleCompare}
-                  >
-                    <GitCompareIcon className="w-4 h-4 mr-4" />
-                    <span>{t("compare")}</span>
-                  </DropdownMenuItem>
-
-                  <DropdownMenuItem
-                    className="cursor-pointer"
-                    onClick={() => setIsSelectingRegion(!isSelectingRegion)}
-                  >
-                    <TextCursorInputIcon className="w-4 h-4 mr-4" />
-                    <span>{t("selectRegion")}</span>
-                  </DropdownMenuItem>
-                </>
-              )
-            }
+            {layout?.name === "sm" && (
+              <>
+                <DropdownMenuItem
+                  className="cursor-pointer"
```

Comment:
should add `id="media-compare-button"` here too. Otherwise it won't work in `sm` layout.